packages/react-components/src/components/ChatMessage/MyMessageComponents/ChatMessageComponentAsRichTextEditBox.tsx

```diff
@@ -153,6 +161,28 @@ export const ChatMessageComponentAsRichTextEditBox = (
     return locale.sendBox;
   }, [/* @conditional-compile-remove(rich-text-editor) */ locale.richTextSendBox, locale.sendBox]);
 
+  /* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */
+  const onCancelAttachmentUpload = useCallback(
+    (attachmentId: string) => {
+      setAttachmentMetadata(attachmentMetadata?.filter((attachment) => attachment.id !== attachmentId));
+    },
+    [attachmentMetadata]
+  );
+
+  /* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */
+  const onRenderAttachmentUploads = useCallback(() => {
+    return (
+      <Stack className={attachmentUploadCardsStyles}>
+        <FluentV9ThemeProvider v8Theme={theme}>
```

Comment:
Just wondering, why do we need to wrap up the _AttachmentUploadCards in it's own theme?

Reply:
This is requirement for the new file cards, otherwise the styles are not applied. This follows what is done in the RichTextSendbox